osgi/karaf-features/src/main/features/features.xml

```diff
@@ -13,17 +13,15 @@
     <feature name="hibernate-validator" version="${project.version}">
         <bundle>mvn:org.hibernate/hibernate-validator/${project.version}</bundle>
         <bundle>mvn:javax.validation/validation-api/${bv.api.version}</bundle>
-
-        <!--
-            As of JBoss Logging 3.2.1.Final, there is non-optional dependency to some Log4j 2 packages; Therefore we
-            are overriding the Import-Package header to make this optional
-        -->
-        <bundle>wrap:mvn:org.jboss.logging/jboss-logging/${jboss.logging.version}$overwrite=merge&amp;Bundle-SymbolicName=org.hibernate.org.jboss.logging&amp;Import-Package=org.apache.log4j;resolution:=optional,org.apache.logging.log4j;version="[2.0,3)";resolution:=optional,org.apache.logging.log4j.message;version="[2.0,3)";resolution:=optional,org.apache.logging.log4j.spi;version="[2.0,3)";resolution:=optional,org.jboss.logmanager;version="[1.5,2)";resolution:=optional,org.slf4j;version="[1.7,2)";resolution:=optional,org.slf4j.spi;version="[1.7,2)";resolution:=optional</bundle>
         <bundle>mvn:com.fasterxml/classmate/${classmate.version}</bundle>
         <bundle>mvn:javax.el/javax.el-api/${javax.el.version}</bundle>
         <bundle>mvn:org.glassfish.web/javax.el/${javax.el.version}</bundle>
-        <bundle>wrap:mvn:com.thoughtworks.paranamer:paranamer:${paranamer.version}</bundle>
+        <bundle>mvn:org.jboss.logging/jboss-logging/${jboss.logging.version}</bundle>
         <bundle>org.jsoup:jsoup:${jsoup.version}</bundle>
```

Comment:
While you are at it, could you add the "mvn:" prefix to the last to ones? Apparently Karaf 3.x didn't complain, but in Karaf 4 I am getting an error due to this when trying to install the feature.